### ISSUE TITLE
Some questions about the test process on unseen object detection.

### ISSUE THREAD

**Issue by interstate50** [salman-h-khan/PL-ZSD_Release#8]
Hi, I'm interested in your paper and try to reproduce your work by Pytorch, but I meet the problem that in your paper **the unseen objects' probability is calculated by W_u, W_s and seen objects' probability($P_u = W_uW_s^Tσ(δ(W_sMD)f)$, equation 9)，how to keep the probability in the range of [0, 1]?** In my work, many unseen objects probabilities are  large and some are even larger than 1. Though I still try to solve this problem, I hope to get your some advice and help. Thanks！

**Comment by shafin5** [comment]
I suspect you are not modifying the word vectors based on vocabulary metric. As illustrated in Fig. 5 in the paper, please apply δ(WMD) after finishing the training. Here, W should be the seen and unseen word2vec inside MSCOCO/word_w2v.txt. Then, please use the modified word vectors to get the unseen probability $P_u = W_uW_s^Tσ(δ(W_sMD)f)$
